ykrt/src/compile/jitc_yk/jit_ir/mod.rs

```diff
@@ -133,6 +133,9 @@ pub(crate) struct Module {
     guard_info: Vec<GuardInfo>,
     /// Indirect calls.
     indirect_calls: Vec<IndirectCallInst>,
+    // FIXME: no pub
+    pub loop_start_vars: Vec<Operand>,
```

Comment:
Let's fix this now, otherwise it'll be forgotten.

Reply:
[515e055](https://github.com/ykjit/yk/pull/1381/commits/515e055862595b4deeafccedb97181ac9f7ce829)